pkgs/development/python-modules/jsonschema/default.nix

```diff
@@ -1,31 +1,51 @@
-{ stdenv, buildPythonPackage, fetchPypi, python
-, nose, mock, vcversioner, functools32 }:
+{ lib
+, buildPythonPackage
+, fetchPypi
+, attrs
+, pyrsistent
+, setuptools
+, six
+, functools32
+, setuptools_scm
+, perf
+, twisted
+, python
+}:
 
 buildPythonPackage rec {
   pname = "jsonschema";
-  version = "2.6.0";
+  version = "3.0.1";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "00kf3zmpp9ya4sydffpifn0j0mzm342a2vzh82p6r0vh10cg7xbg";
+    sha256 = "0c0a81564f181de3212efa2d17de1910f8732fa1b71c42266d983cd74304e20d";
   };
 
-  checkInputs = [ nose mock vcversioner ];
-  propagatedBuildInputs = [ functools32 ];
+  buildInputs = [
+    setuptools_scm
+  ];
 
-  postPatch = ''
-    substituteInPlace jsonschema/tests/test_jsonschema_test_suite.py \
-      --replace "python" "${python.pythonForBuild.interpreter}"
-  '';
+  propagatedBuildInputs = [
+    attrs
+    pyrsistent
+    setuptools
+    six
+    functools32
+  ];
+
+  checkInputs = [
+    twisted
+    perf
+  ];
 
   checkPhase = ''
-    nosetests
+    ${python.interpreter} setup.py test --test-suite=jsonschema.tests
   '';
 
-  meta = with stdenv.lib; {
-    homepage = https://github.com/Julian/jsonschema;
+  meta = with lib; {
     description = "An implementation of JSON Schema validation for Python";
+    homepage = https://github.com/Julian/jsonschema;
     license = licenses.mit;
-    maintainers = with maintainers; [ domenkozar ];
+    maintainers = [ maintainers.costrouc ];
```

Comment:
is @domenkozar ok with the change ?

Reply:
Whoops that was not intentional. I have a tool that auto generates the derivations and I forgot to add him back in. I will add him back

Reply:
Fixed.